webpack.config.js

```diff
@@ -21,13 +21,44 @@ const extractSass = new ExtractTextPlugin({
 	allChunks: true
 });
 
+const ENTRYPOINTS = {
+	OneNotePicker: `exports/oneNotePicker.Index`,
```

Comment:
I think we should lowercase the i in Index to match our file naming convention

Reply:
This requires changes in this file, as well as the naming of the files themselves